qiskit_aqua/algorithms/classical/svm/svm_classical.py

```diff
@@ -60,12 +65,17 @@ def __init__(self, configuration=None):
 
     def init_params(self, params, algo_input):
         SVM_Classical_params = params.get(QuantumAlgorithm.SECTION_KEY_ALGORITHM)
+
+        multiclass_extension_params = params.get(QuantumAlgorithm.SECTION_KEY_MULTICLASS_EXTENSION)
```

Comment:
I think this piece of codes should be only executed if it is multiclass.

Reply:
done